### PR TITLE
Add unit test runner script. Fixed some errors caught by unit tests

### PR DESCRIPTION
**Pull request by davidz627**
Part 1 of: #49 
Part 2 will be adding run-unit.sh to the test-infra config

/assign @msau42

### CODE REVIEW

File: pkg/gce-pd-csi-driver/controller.go [hunk 1]
```diff
@@ -52,23 +52,38 @@ const (
 	attachableDiskTypePersistent = "PERSISTENT"
 )
 
-func getRequestCapacity(capRange *csi.CapacityRange) (capBytes int64) {
+func getRequestCapacity(capRange *csi.CapacityRange) (int64, error) {
```
Comment (msau42) [inline pkg/gce-pd-csi-driver/controller.go]: You may want to check out what I did for this (in fact, it should probably be a utility function that can be shared by multiple drivers), and my test cases.

Comment (davidz627) [reply pkg/gce-pd-csi-driver/controller.go]: I wrote some tests for this function in a followup PR. Lets work on a canonical `getRequestCapacity`.

**Pull request merged**